plugins/inputs/suricata/README.md

```diff
@@ -94,6 +98,19 @@ All fields are numeric.
     - tcp_synack
     - ...
 
+Some fields of the Suricata alerts are strings, for example the signatures. See https://suricata.readthedocs.io/en/suricata-6.0.0/output/eve/eve-json-format.html?highlight=priority#event-type-alert for more information.
+
+- suricata_alert
+  - fields:
+    - action
+    - gid
+    - severity
+    - signature
```

Comment:
Does it make sense to convert `severity` and maybe also `action` to a tag? You could add a
```toml
as_tag = ["action", "severity"]
```
list option for this to allow the user to make this choice...

Reply:
That would be very useful, working on it. Thanks

Reply:
In some cases severity is not defined, I would not add this (like the current stats approach) to prevent errors

Reply:
Can be done by processors later on.